examples/hbd.cpp

```diff
@@ -0,0 +1,112 @@
+// CPP program to print Happy Birthday
+#include<iostream>
+using namespace std;
+
+int main()
+{
+    // Print first row
+    char ch = '*';
+    for(int i=1; i<=34; i++)
+    {
+        if (i==5||i==7||i==10||i==11||i==14||i==15||
+            i==16||i==18||i==19||i==20||i==22||i==24)
+            cout << ch<< ch ;
+        else cout << "  " ;
+        
+    }
+    
+    // Print second row
+    cout << endl;
+    for(int i=1; i<=34; i++)
+    {
+        if(i==5||i==7||i==9||i==12||i==14||i==16
+           ||i==18||i==20||i==22||i==24)
+            cout << ch<< ch ;
+        else cout << "  " ;
+        
```

Comment:
@anu-shree Please remove this blank line. 
Check the CodeFactor checks [here](https://www.codefactor.io/repository/github/vinitshahdeo/hbd/pull/13).